src/System.Private.ServiceModel/src/System/ServiceModel/Channels/TransportDefaults.cs

```diff
@@ -140,8 +140,8 @@ public static class BinaryEncoderDefaults
 
     public static class TransportDefaults
     {
-        public const bool ExtractGroupsForWindowsAccounts = true;
         //public const bool ExtractGroupsForWindowsAccounts = SspiSecurityTokenProvider.DefaultExtractWindowsGroupClaims;
```

Comment:
The default value hasn't changed. The reason why we had this is because SspiSecurityTokenProvider wasn't ported in the initial cut of code. Now that it's in we can revert back to having the correct  reference to SspiSecurityTokenProvider